cf-agent/cf-agent.c

```diff
@@ -191,6 +192,7 @@ static const char *const HINTS[] =
 {
     "Bootstrap CFEngine to the given policy server IP, hostname or :avahi (automatic detection)",
     "Set or override bundlesequence from command line",
+    "Override the work directory for testing (same as setting CFENGINE_TEST_OVERRIDE_WORKDIR)",
```

Comment:
I would change this to `Override the default /var/cfengine work directory...`, just so that there is no doubt what is being talked about.